skyportal/app_server.py

```diff
@@ -47,7 +47,7 @@ def make_app(cfg, baselayer_handlers, baselayer_settings):
     handlers = baselayer_handlers + [
         # API endpoints
         (r'/api/comment(/[0-9]+)?', CommentHandler),
-        (r'/api/comment(/[0-9]+)/(download_attachment)', CommentHandler),
+        (r'/api/comment(/[0-9]+)/(attachment)', CommentHandler),
```

Comment:
Should this be CommentAttachmentHandler, and `(attachment)` change to `attachment`?

Reply:
Ah, yes! Good catch -- this was an unsuccessful merge conflict resolution (from your PR that re-ordered these). Looks like this is why some of the tests are failing, too.